package.json

```diff
@@ -40,7 +40,7 @@
     "*.{js,ts,json}": "eslint"
   },
   "dependencies": {
-    "@bitcoin-js/tiny-secp256k1-asmjs": "^2.2.3",
+    "@bitcoinerlab/secp256k1": "^1.2.0",
```

Comment:
I wonder if https://github.com/bitcoinjs/tiny-secp256k1 would make more sense. It may be a breaking change because it requires WASM, but how many platforms that support BigInt don't support WASM?